data_steward/utils/participant_summary_requests.py

```diff
@@ -40,6 +49,18 @@
 to the Curation naming convention in the `get_site_participant_information` function
 """
 
+FIELDS_OF_INTEREST_FOR_DIGITAL_HEALTH = [
+    'participantId', 'digitalHealthSharingStatus'
+]
+
+BASE_URL = 'https://{api_project_id}.appspot.com/rdr/v1/ParticipantSummary'
+
+# Retry parameters
+STATUS_FORCELIST = [500, 502, 503, 504]
```

Comment:
this is a good list.  what happens if we get a 404?  does it retry or just fail?  if i'm reading docs right, it just fails.  which is good.